pkg/kubernetessecrets/kubernetes_secrets_builder.go

```diff
@@ -120,7 +120,7 @@ func BuildKubernetesSecretData(fields []model.ItemField, files []model.File) map
 	for _, file := range files {
 		content, err := file.Content()
 		if err != nil {
-			log.Error(err, "Could not load contents of file %s", file.Name)
+			log.Error(err, fmt.Sprintf("Could not load contents of file %s", file.Name))
```

Comment:
`[non-blocking heads-up]`

I've noticed a similar pattern in `pkg/onepassword/secret_update_handler.go`. Coincidentally, for those the number or arguments is even, but it doesn't seem to be the desired outcome. Would you be kind and address those as part of this PR? Addressing them in a separate PR also works.

Reply:
Fixed this in followup commit